demisto_sdk/commands/generate_docs/common.py

```diff
@@ -147,7 +147,7 @@ def generate_table_section(
     for item in data:
         tmp_item = "    |" if numbered_section else "|"
         escape_less_greater_signs = (
-            "First fetch time" in item
+            "First fetch time" or "First fetch timestamp" in item
```

Comment:
note that `First fetch time` is a substring of `First fetch timestamp` - what does this change do?

Reply:
yeah i thought that this logic applies, turned out no, it checks if the string is a key in the dict, not a substring in a string

Reply:
In this case, I'd suggest `if {"First fetch time", "First fetch timestamp"}.issubset(item.keys())`

Reply:
as seen in the screenshot, the mentioned logic did not work
<img width="613" alt="Screen Shot 2023-05-18 at 15 48 20" src="https://github.com/demisto/demisto-sdk/assets/49760643/17f64da2-7075-4c72-bf6c-073b64ddecd7">

the change i made checks either key in the dict.

Reply:
my bad - `intersection`, not `issubset`.